docs/source/update_project.rst

```diff
@@ -0,0 +1,77 @@
+Keeping your project up to date
+===============================
+
+Once your project is under version control you'll be able to keep your project up 
+to date as new features are added to the template by running the following command:
+
+.. code-block:: bash
+
+    >> copier update
+
+Copier will automatically check to see if a newer version of the original template 
+is available and if so the changes will be automatically applied. Neato!
+
+And of course, because your project is under version control, if you don't like 
+the new changes, you can always revert back to the previous state.
```

Comment:
Maybe also add something *here* about checking the diffs on conflicting or overwritten files.

I find that I have a bunch of `.rej` files that indicate something was conflicting that I have to clean up (but in the process of cleaning them up, it reminds me to double-check the new contents of the files).